PyPI tarball missing `kernel_interface.cpp`

Hey! I tried running `pip install george` on an ARM system and I saw this error: 

```
      -- Configuring done (5.8s)
      CMake Error at /tmp/test/lib/python3.11/site-packages/pybind11/share/cmake/pybind11/pybind11Tools.cmake:181 (add_library):
        Cannot find source file:
          src/george/kernel_interface.cpp
      Call Stack (most recent call first):
        CMakeLists.txt:12 (pybind11_add_module)
```
When I download from the repo and build from source, it worked just fine. When I inspected the tarball from PyPi, I noticed the file is indeed missing. I also confirmed the issue existed on x86; I ran `pip install george --no-binary=george` and saw the same error message.

I don't know why the PyPi tarball is missing this .cpp file, but hopefully adding it will fix the issue.

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -86,6 +86,10 @@ jobs:
         run: python -m build --sdist .
       - name: Check the sdist
         run: python -m twine check dist/*.tar.gz
+      - name: Check sdist build
+        run: |
+          python -m pip install dist/*.tar.gz
+          python -c "import george"
       - uses: actions/upload-artifact@v4
         with:
           name: sdist

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -15,7 +15,11 @@ build-backend = "scikit_build_core.build"
 
 [tool.scikit-build]
 metadata.version.provider = "scikit_build_core.metadata.setuptools_scm"
-sdist.include = ["src/george/george_version.py"]
+sdist.include = [
+    "src/george/george_version.py",
+    "src/george/kernel_interface.cpp",
+    "src/george/solvers/_hodlr.cpp",
+]
 wheel.install-dir = "george"
 minimum-version = "0.5"
 build-dir = "build/{wheel_tag}"

